cfg/aesbackend_test.go

```diff
@@ -0,0 +1,95 @@
+/*
+ * knoxite
+ *     Copyright (c) 2020, Nicolas Martin <penguwin@penguwin.eu>
+ *
+ *   For license see LICENSE
+ */
+package cfg
+
+import (
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+const testPassword = "test"
+
+func TestAESBackendLoad(t *testing.T) {
+	cwd, _ := os.Getwd()
+
+	// try to load the config from an absolute path using a URI
+	u, _ := url.Parse("crypto://" + testPassword + "@" + filepath.Join(cwd, "testdata", "knoxite-crypto.conf"))
```

Comment:
We need to urlencode & -decode all paths & passwords in these situations!

Reply:
Thats right - I've now fixed it